Mathmagician.Tests/FibonacciTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class FibonacciTests
+    {
+        [TestMethod]
+        public void GenerateInstanceOfFibonacci()
+        {
+            Fibonacci fibonacci = new Fibonacci();
+            Assert.IsNotNull(fibonacci);
+        }
+        [TestMethod]
+        public void GenerateANumber()
+        {
+            Fibonacci new_fib = new Fibonacci();
+            new_fib.GenerateFibonacci(1);
+            int fibToCheck = new_fib.sequence[0];
+            Assert.IsNotNull(fibToCheck);
+        }
+        [TestMethod]
+        public void WhatHappensIfYouEnterZero()
+        {
+            Fibonacci fibWithZero = new Fibonacci();
+            fibWithZero.GenerateFibonacci(0);
+            //the array already contains the first two numbers, 1 1
+            //this assert makes sure no new numbers were added.
+            Assert.IsTrue(fibWithZero.sequence.Count == 2);
```

Comment:
I got that taken care of with another if statement to check if the user asks for less than 3.
if - the user asks for 0, it prints a string and closes
else if -the user asks for less than 3, it prints '1' once or twice depending
else- it prints 1 twice, then it starts making fibzos and printing those, adding the new ones to the list.

I went back in and edited it to be more straightforward. Also added a test to make sure that inputting '1' gets you '1' back